content/doc/book/using/best-practices.adoc

```diff
@@ -0,0 +1,190 @@
+To enforce consistent naming conventions across all projects, you can enable the "Restrict project naming" setting in the system configuration. This will ensure that naming restrictions are enforced uniformly.
+
+=== Fingerprint your dependencies
+
+When dealing with interdependent projects, it can be challenging to keep track of which version of one project is used by another.
+However, Jenkins offers a solution called "file fingerprinting" to simplify this process.
+
+More information is available on the link:/doc/book/using/fingerprints/[fingerprinting page].
+
+=== Don't use the Maven job type
+
+Jenkins has been providing the plugin:maven-plugin[Maven integration plugin] for many years, allowing users to create Maven projects using the "Maven project" selection from the Jenkins "New item" menu.
+While the Maven job type offers a higher level of integration with Maven builds, it can sometimes introduce unnecessary complexities due to this deep integration.
+
+Consider using organization folders, multibranch Pipelines, or Pipeline jobs instead of the Maven job type. These alternatives provide more flexibility and simplicity in managing your Jenkins jobs and workflows.
+
+The Jenkins project uses organization folders to build link:https://ci.jenkins.io/job/Core/[Jenkins core] and link:https://ci.jenkins.io/job/Plugins/[Jenkins plugins] on ci.jenkins.io.
+A Jenkins Pipeline builds Maven projects easily and provides much better control for Maven users.
+
+Refer to the link:https://plugins.jenkins.io/maven-plugin/#plugin-content-risks[Maven plugin documentation] for more details.
+
+== Manage your controller
+
+The Jenkins controller plays a crucial role as a central resource and requires effective management for optimal performance.
+By following these practices, you can ensure that your controller provides the best possible experience for users.
+
+=== Secure the controller
+
+Jenkins installations come with security enabled by default, which is a crucial aspect for protecting your system.
+While it is technically possible to disable security, it is strongly advised **not** to do so.
+Disabling security can leave your Jenkins instance vulnerable to unauthorized access and potential security breaches.
+It is important to maintain a secure environment by keeping security enabled at all times.
+
+Details are in the link:/doc/book/security/securing-jenkins/[securing Jenkins] chapter of the User Handbook.
+
+=== Back up regularly
+
+Even the most reliable systems can experience failures. 
+That's why it's crucial to be prepared and regularly check the health of your backups.
+Backups are a critical component of ensuring the integrity and availability of your data.
+Regularly testing your backups and verifying their completeness and restorability will help you mitigate the impact of any potential failures and ensure that your data can be recovered effectively when needed.
```

Comment:
```suggestion
Regularly testing your backups and verifying their completeness and restorability, will help you mitigate the impact of any potential failures and ensure that your data can be recovered effectively when needed.
```
adding a comma since this is a longer sentence

Reply:
I disagree with the comma inserted here.